### PR TITLE
Added hw03 branch and files

### PR DESCRIPTION
**Pull request by diegodulanto**

### CODE REVIEW

**Comment by iannel** [comment]
Hi Diego. I didn't receive a pull request from you yet, but I'm going to go ahead and review your code.

File: hw03/hw03.Rmd [hunk 1]
```diff
@@ -0,0 +1,252 @@
...
+
+  1. Inspect the item responses (e.g., with graphs or by summarizing distinct values). 
+     Is anything unusual?
+```{r}
+green_data %>% 
+  summarise(
+    across(
+      c(green1:student),
+      list(Max = ~ max(.x, na.rm = TRUE),
+           Min = ~ min(.x, na.rm = TRUE))))
```
Comment (iannel) [inline hw03/hw03.Rmd:46]: I think this can be condensed by using ```summary(data)```

File: hw03/hw03.Rmd [hunk 2]
```diff
@@ -0,0 +1,252 @@
...
+
+filter(green_data, id == 5549)
```
Comment (iannel) [inline hw03/hw03.Rmd]: You could use comments here to tell the reader what you are doing.

File: hw03/hw03.Rmd [hunk 3]
```diff
@@ -0,0 +1,252 @@
...
+
+green_data %>% 
+  ggplot() +
+  geom_jitter(aes(x = "id", y = "green"))
+
+#There are -99s in the data that needs to be recoded
+
+green_recode <-
+  green_data %>%
+  replace_with_na_all(~.x == -99) %>%
```
Comment (iannel) [inline hw03/hw03.Rmd]: You can also use ```na_if("-99")```

File: hw03/hw03.Rmd [hunk 4]
```diff
@@ -0,0 +1,252 @@
...
+green_reversed <-green_recode %>%
+  mutate(
+    across(
+      all_of(reversed),
+           ~ recode(.x, 
+                    "5" = 1, 
+                    "4" = 2, 
+                    "3" = 3, 
+                    "2" = 4, 
+                    "1" = 5)))
```
Comment (iannel) [inline hw03/hw03.Rmd:86]: Very neat code. You could also use ```~6 - .x```. Also, you may want to change the names of your recoded items

File: hw03/hw03.Rmd [hunk 5]
```diff
@@ -0,0 +1,252 @@
...
+                         to=c(0,100), 
+                         from=c(1,5)),
+    comp_pomp = rescale(comp_tot, 
+                        to=c(0,100), 
+                        from=c(1,5)),
+    intel_pomp = rescale(intel_tot, 
+                         to=c(0,100), 
+                         from=c(1,5)),
+    open_pomp = rescale(open_tot, 
+                        to=c(0,100),
```
Comment (iannel) [inline hw03/hw03.Rmd:122]: Good job!

File: hw03/hw03.Rmd [hunk 6]
```diff
@@ -0,0 +1,252 @@
...
+                            color=student)) + 
+  theme_bw() +
+  geom_point(alpha = .5) +
+  geom_smooth(aes(group=student))
+
+grid.arrange(comp_scatter, 
+             intel_scatter, 
+             open_scatter)
+
+## The overall relationships between green reputation and personalities are shared with students and nonstudents. Higher the score on compassion, intellect, and openness, higher the green reputation. However, for students, intellect and openness have a stronger effect on the green reputation.
```
Comment (iannel) [inline hw03/hw03.Rmd]: Nice job!

**Review by iannel — COMMENTED** [review]
Great job! Make sure you create a "tidy" submission before submission

**Comment by diegodulanto** [comment]
@khoanguyen0812 @iannel thank you for reviewing my code, Ian!

**Pull request merged**